packages/front-end/pages/features/[fid].tsx

```diff
@@ -43,6 +45,9 @@ export default function FeaturePage() {
   const [lastDisplayedVersion, setLastDisplayedVersion] = useState<
     number | null
   >(null);
+  const [diagnosticsResults, setDiagnosticsResults] = useState<Array<
+    FeatureEvalDiagnosticsQueryResponseRows[number] & { id: string }
+  > | null>(null);
```

Comment:
question: Why is this state here? Can we move it inside `FeatureDiagnostics` instead?